packages/apsara-ui/src/Notification/Notification.tsx

```diff
@@ -31,52 +31,64 @@ export const useNotification = () => {
 };
 
 export const NotificationProvider = ({ children }: any) => {
-    const [toasts, setToasts] = useState<Notification[]>([]);
-    
-    const showNotification = useCallback((toast: Notification) => {
-        setToasts([...toasts, { ...toast, id: uuid() }]);
-    }, [toasts, setToasts]);
+    const [queue, setQueue] = useState<Notification[]>([]);
```

Comment:
since it is a Notification, let's just call it `notifications`?

```suggestion
    const [notifications, setNotifications] = useState<Notification[]>([]);
```